### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by kkirilov1** [community/community#102526] [GitHub Education]
### Hi everyone,

Good afternoon,

I applied for GitHub Student Developer Pack almost a week ago, but unfortunately all I can see is this message:

**We're still processing your application. Once this is complete you will have access to the student developer pack offers.**

There's no other information provided.  
Any idea on how long does it normally take to be approved and get full access to the Student developer pack?
Thanks in advance.

Regards

**Comment by faridnec** [comment]
Hey I think it varies for each applicant which in my case I got it like within 3 days. I think you can wait and also trying to reach them out via Github Edu Support [here](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket)
